test/test_prototype_transforms_kernels.py

```diff
@@ -0,0 +1,394 @@
+import functools
+import itertools
+import math
+
+import numpy as np
+import pytest
+import torch.testing
+import torchvision.prototype.transforms.functional as F
+from common_utils import cpu_and_gpu, needs_cuda
+from datasets_utils import combinations_grid
+from prototype_common_utils import ArgsKwargs, assert_close, make_bounding_box_loaders, make_image_loaders
+from torchvision.prototype import features
+
+
+class KernelInfo:
+    def __init__(
+        self,
+        kernel,
+        *,
+        sample_inputs_fn,
+        reference=None,
+        reference_inputs_fn=None,
+        **closeness_kwargs,
+    ):
+        self.kernel = kernel
+        # smoke test that should hit all valid code paths
+        self.sample_inputs_fn = sample_inputs_fn
+        self.reference = reference
+        self.reference_inputs_fn = reference_inputs_fn or sample_inputs_fn
+        self.closeness_kwargs = closeness_kwargs
+
+    def __str__(self):
+        return self.kernel.__name__
+
+
+def pil_reference_wrapper(pil_kernel):
```

Comment:
The reference defined in the `KernelInfo` will be passed the same inputs as the kernel. Since we use the PIL kernel as reference for its tensor counterpart, this is simple wrapper to avoid defining the same kind of reference function over and over.